react-frontend/src/listings/ListingView.js

```diff
@@ -65,6 +64,7 @@ function ListingView() {
         const response = await axios.get(
           process.env.REACT_APP_BACKEND_LINK + `/listings/${listingID}`,
         );
+        (console.log(response.data[0].category))
```

Comment:
Consider removing console.log statements at some point. so that terminal doesn't get bombed when other people are writing debugging statements like that. Up to you